addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6879,6 +6879,7 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
      */
     updateUI() {
         if (this.rerender) {
+            this.trigger_up('update_color_previews');
```

Comment:
I don't think this is the right, I am not sure to understand the first paragraph of your commit message but the second one says "ensures that when we select a snippet or access the "Blocks" tab, the color options displayed accurately reflect the current theme color" -> why would we need to update the color displayed each time we select a snippet or access the "Blocks" tab? From what I understand of the code, we update the CSS color variables each time the bundles are updated (`_refreshBundles`)... that should be enough? Why is not it?

Reply:
Hello @qsm-odoo ,
Done the changes.
Thanks.

Reply:
Continuing the discussion via a new review...